packages/now-cli/src/util/projects/link.ts

```diff
@@ -43,11 +36,21 @@ export async function getLinkedProject(
   path: string
 ): Promise<[Org | null, Project | null]> {
   try {
-    const json = await readFile(join(path, NOW_FOLDER, NOW_PROJECT_LINK_FILE), {
-      encoding: 'utf8',
-    });
+    let link: ProjectLink;
+
+    if (process.env.NOW_ORG_ID && process.env.NOW_PROJECT_ID) {
```

Comment:
Perhaps destructure these env vars first since they're accessed immediately within the `if` block?

Reply:
Sure 👍